### ISSUE TITLE
Full-page / responsive width for boundary? 

### ISSUE THREAD

**Issue by TD-Stanley** [Foliotek/Croppie#214]
Is it possible to set the boundary and view port to the full width of the screen, in order to implement twitter-like, profile-header upload?

**Comment by thedustinsmith** [comment]
Sure - if you get the full width of the screen in your js, just initialize your croppie with that number.  

If you're referring to the window resizing and resizing your croppie, then no, there's no good way to do that.  You can destroy and reinitialize your croppie, but that's probably inefficient.

Something like this should work:

```
var w = $(window).width(),
    h = $(window.height();
$('.my-el').croppie({
   boundary: { width: w, height: h },
   viewport: { width: w, height: h }
});
```

**Comment by TD-Stanley** [comment]
Yeah, I had come up with something similar. But, was trying to figure out how to implement it with window resize
